lib/qesapdeployment.pm

```diff
@@ -295,14 +349,17 @@ sub qesap_prepare_env {
     my $variables = $args{openqa_variables};
     my $provider = $args{provider};
     my %paths = qesap_get_file_paths();
-    my $qesap_conf_src = "sles4sap/qe_sap_deployment/" . $paths{qesap_conf_filename};
 
-    qesap_create_folder_tree();
-    qesap_get_deployment_code();
-    qesap_pip_install();
+    # Option to skip straight to configuration
+    unless ($args{only_configure}) {
+        qesap_create_folder_tree();
+        qesap_get_deployment_code();
+        qesap_pip_install();
+
+        record_info("QESAP yaml", "Preparing yaml config file");
+        assert_script_run('curl -v -L ' . $paths{qesap_conf_src} . ' -o ' . $paths{qesap_conf_trgt});
```

Comment:
**-f** (it was there in the original code, any specific reason not to have it?)

Reply:
Ah sorry... just a honest mistake while replacing the qesap_conf_src variable. Will fix it.